dashboard/modules/reporter/reporter_head.py

```diff
@@ -23,9 +27,34 @@
 from ray._private.usage.usage_constants import CLUSTER_METADATA_KEY
 from ray.autoscaler._private.commands import debug_status
 
+from ray.util.state.common import (
+    ListApiOptions,
+)
+from ray.dashboard.state_aggregator import StateAPIManager
+from ray.util.state.state_manager import (
+    StateDataSourceClient,
+)
+
 logger = logging.getLogger(__name__)
 routes = dashboard_optional_utils.ClassMethodRouteTable
 
+EMOJI_WARNING = "&#x26A0;&#xFE0F;"
+WARNING_FOR_MULTI_TASK_IN_A_WORKER = "Warning: This task is running in a worker process that is running multiple tasks. The information that follows may come from any of these tasks:"
+
+
+def get_warning_text_in_html(task_ids: List[str]) -> str:
+    return '<p style="color: #E37400;">{} {} </br> </p> </br>'.format(
+        EMOJI_WARNING, WARNING_FOR_MULTI_TASK_IN_A_WORKER + str(task_ids)
```

Comment:
just nit: i don't think you need a global var here. it makes it harder to read actually?

Reply:
and maybe we could move this to where this is used?

Reply:
That's a good idea!